### ISSUE TITLE
A user is shown a random photo from the database to classify

### ISSUE THREAD

**Issue by wrflemin** [CMPUT401Team/CMPUT401Project#32]

**Comment by wrflemin** [comment]
The random image Api is ready in the branch: displayRandomPhoto
to call this endpoint you can just go http://localhost:9000/api/v1/randomImage with the above api key value pair in the header and it will return json in the form {“id”: 123, “link”:”http//awsImageUrl.com”} where link is the direct link to the image that is hosted by AWS.

**Comment by to4ka5** [comment]
this is good with one caveat: I think in the majority of cases we'll be showing a short sequence of images (like a mini slide show) similar to how it's done in Snapshot Wisconsin with 3 images in a row on each viewing. So maybe this API could be implemented in terms of sequences rather than a single image, i.e., returning an array of objects with id and link. I didn't do my part with the database yet, though, so there aren't tables to select sequences from, so let's leave it as is for now.

Could you please open a pull request in Bitbucket so I could review/add some comments? It doesn't let me add comments on the branch directly without the pull request.

**Comment by to4ka5** [comment]
oops, accidentally closed the issue instead of just commenting

**Comment by wrflemin** [comment]
Good point! This API will definitely return a sequence of images in the future, so I will make an update to it on Thursday to reflect that behaviour. That way the front end isn't surprised when we change it. I also made a pull request for this feature.
